dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/core/jackson/ModelResolver.java

```diff
@@ -1153,9 +1153,10 @@ protected Discriminator resolveDiscriminator(JavaType type, ModelConverterContex
                 DiscriminatorMapping mappings[] = directSchemaAnnotation.discriminatorMapping();
                 if (mappings != null && mappings.length > 0) {
                     for (DiscriminatorMapping mapping : mappings) {
-                        if (!mapping.value().isEmpty() && !mapping.schema().equals(Void.class)) {
-                            discriminator.addMapping(mapping.value(), constructRef(((SchemaImpl) context.resolve(mapping.schema())).getName()));
-                        }
+                        //@DiscriminatorMapping is not specified as default value anywhere - so it must be user-specified, hence the following if-check is not needed.
```

Comment:
What if user specifies `@Schema(discriminatorMapping={@DiscriminatorMapping()})`
It still be Void.class

Reply:
If the user does that then I think it should be considered a user error and the validator should output a message regarding it (i.e. empty discriminator mapping value with Void.class), which is probably useful to them as well instead of us simply ignoring the invalid value.

Reply:
Sounds good, we need to make sure that no NPE will happen if we resolve Void.class

Reply:
Good point. Seems the discriminator mapping handling within Schema is not complete. So opened an issue to verify it once that's done.